sql-plugin/src/main/scala/com/nvidia/spark/rapids/spill/SpillFramework.scala

```diff
@@ -452,26 +490,62 @@ class SpillableDeviceBufferHandle private (
     materialized
   }
 
+  private var toSpill: Option[DeviceMemoryBuffer] = None
```

Comment:
Same comment as above. I just got a bit confused. It didn't take long to figure it out, but a comment would be nice.